### ISSUE TITLE
Paper-buttons-row not working in popup unless visited first

### ISSUE THREAD

**Issue by Alfiegerner** [thomasloven/hass-browser_mod#119]
Hi,

I'm looking to move on from button-entity-row as it stopped working in 113.  Currently looking at using paper-buttons-row in my browser_mod pops and more-info replacement, however the button presses do not work unless I visit a paper buttons populated card in vanilla lovelace first.

After visiting card with same type in vanilla dashboard, everything works fine in browser_mod panels.

Any ideas on what I could try to get around this?  Also asking for assistance on paper-buttons-row but thought to ask here in case anyone has any ideas.

Thanks,

Alex

**Comment by thomasloven** [comment]
Are you talking about a different custom card?

The problem is probably that the paper-buttons-row element is not loaded in by default.
You can make a card load in by using the cardHelpers. Something like:
```js
window.loadCardHelpers.then(cardHelpers => {
  cardHelpers.createCardElement({type: your-card});
});
```
should be enough.

**Comment by Alfiegerner** [comment]
I think we got to the bottom of it, we found that paper-buttons-row was not loading an action-handler and as none were present in the popup no actions worked. This is as it is a row based element and entities card doesn't load action handler (unless special row buttons used).

Workaround is to add card with action handler to vertical stack in popup and author working on adding action handler to row element.

Thanks!
